Kickstarter-iOS/Views/Controllers/ProjectNavigatorViewController.swift

```diff
@@ -169,6 +173,8 @@ extension ProjectNavigatorViewController: ProjectPamphletViewControllerDelegate
     _: ProjectPamphletViewController,
     panGestureRecognizerDidChange recognizer: UIPanGestureRecognizer
   ) {
+    if #available(iOS 13.0, *) { return }
```

Comment:
I considered placing this check in the VM and trying to test it but we ran into a couple of issues trying to test these availability checks in the past and removed it in #731 so I'm thinking this is ok for this?

Reply:
I think that in this case it's fine to have the check here.